src/ucp/wireup/wireup_ep.c

```diff
@@ -471,7 +471,7 @@ UCS_CLASS_INIT_FUNC(ucp_wireup_ep_t, ucp_ep_h ucp_ep)
     self->flags            = 0;
     self->progress_id      = UCS_CALLBACKQ_ID_NULL;
     ucs_queue_head_init(&self->pending_q);
-
+    UCS_BITMAP_CLEAR(&self->cm_resolve_tl_bitmap);
```

Comment:
minor: add an empty line after
```suggestion
    UCS_BITMAP_CLEAR(&self->cm_resolve_tl_bitmap);

```